.travis.yml

```diff
@@ -1,3 +1,10 @@
+matrix:
+  include:
```

Comment:
sadly, not yet: https://docs.travis-ci.com/user/reference/precise/#Virtualization-environments